vir/defs/high/ast/ty.rs

```diff
@@ -11,6 +11,9 @@ pub enum Type {
     /// Mathematical floats that corresponds to Viper's Float.
     MFloat32,
     MFloat64,
+    // TODO: is this okay? remove struct
+    Lifetime(Lifetime),
+    NonameLifetime,
```

Comment:
Why `Lifetime` is not simply an enum?

Reply:
Was deciding between an enum or renaming `NonameLifetime` to `GhostType` such that it could be used for other ghost assignments in future. What do you think?

Reply:
Oh, that is the type of an actual lifetime variable. Then, I would name it `Lifetime` and rename the current `Lifetime` to `LifetimeConst`.

Reply:
yes, the actual lifetime (`Lifetime(Lifetime)` here) is actually not needed as a member of the Type enum. I will remove it. The `NonameLifetime` is currently only needed to construct the `VariableDecl` for `GhostAssignment`. 

`Lifetime -> LifetimeConst` and `NonameLifetime -> Lifetime` sounds good.